### ISSUE TITLE
Training on custom dataset

### ISSUE THREAD

**Issue by GriesserP** [BGU-CS-VIL/DeepDPM#6]
Thank you for your super contributions,
What would be the best way to use your code to train and predict the cluster assignment of a list of embeddings representing my data points. i.e. something like .fit(X) or .fit_predict(X) from sklearn ?

**Comment by meitarronen** [comment]
Thank you for your interest in our work!
For training, you should follow the instructions available in the readme file, and train using the DeepDPM.py file (see section Training in our readme). 
In theory, you could also use this on your test set (DeepDPM is an unsupervised method, labels are not used in any training stage (we use them in our code only for evaluation, but they are not used in training).

That said, we have recently uploaded an example of using DeepDPM for inference (fit_predict). See example script under scripts/DeepDPM_load_from_checkpoint.py
